Makefile

```
@@ -125,7 +125,7 @@ check-no-git-lfs-shell: ## Ensure no code shells out to git-lfs binary (see .cla
 
 test-preflight: lint check-no-git-lfs-shell test-all test-slow ## Pre-PR quality gate: lint + all unit tests + slow tests
 
-test-digital-twin: test-team-context-twin test-ledger-twin
+test-digital-twin: test-ledger-twin ## Digital twin tests (team_context_twin pending, see ox-au5)
```

checkmake (0.3.2)

[warning] Target "test-digital-twin" should be declared PHONY.

(phonydeclared)